lib/db.js

```diff
@@ -54,11 +54,20 @@ class LimitDBRedis extends EventEmitter {
       redisOptions
     };
 
+    this.pingConfig = {
+      enabled: config.ping?.enabled || false,
+      interval: config.ping?.interval || 1000,
+      maxFailedAttempts: config.ping?.maxFailedAttempts || 5,
+      reconnectIfFailed: config.ping?.reconnectIfFailed || false,
```

Comment:
Since we are using a feature flag to control this behavior, this must be evaluated at all time. I would change this into a function () => Bool

Reply:
Totally true. I'll change it. We'll need to somehow receive a feature flag instance, right?. I'd need to dive deeper into that.

Reply:
Sorry, I re-read your comment and I see what you mean now. Receiving a user-defined function to evaluate it. 
Sure thing, I'll do that. Thanks for the comment!